python/tvm/contrib/hexagon/build.py

```diff
@@ -182,7 +182,7 @@ def upload(self, local_path: Union[str, pathlib.Path], remote_filename: str):
         assert self._workspace
         self._copy_to_remote(local_path, os.path.join(str(self._workspace), remote_filename))
 
-    def start_session(self) -> Session:
+    def start_session(self, name="hexagon-rpc") -> Session:
```

Comment:
nit: keep the kwarg named the same so it's easier to analyze as we move up and down the stack. also, can you update the docstring?

Reply:
done.